webdriver-spec.html

```diff
@@ -8194,10 +8194,10 @@
  <li><p>If <var>screenshot result</var> is an <a>error</a>,
   return <var>screenshot result</var>.
 
- <li><p>Let <var>canvas</var> be <var>screenshot result</var>’s data.
-
- <li><p>Let <var>encoding result</var> be the result of
-  <a>encoding as Base64</a> the <var>canvas</var> element.
+ <li><p>If <var>screenshot result</var> is a
+  <a><code>canvas</code> element</a>, let <var>encoding result</var>
+  be the result of <a>encoding as Base64</a> the
+  <var>screenshot result</var>.
```

Comment:
The result is either success with some data or an error. Think of it as the [`Result` enum](https://doc.rust-lang.org/std/result/enum.Result.html) in Rust.

Consequently, saying “if _screenshot result_ is a `<cavnas>` element” doesn’t make sense.